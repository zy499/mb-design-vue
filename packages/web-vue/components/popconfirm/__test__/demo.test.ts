import demoTest from '../../../scripts/demo-test';

demoTest('popconfirm');
