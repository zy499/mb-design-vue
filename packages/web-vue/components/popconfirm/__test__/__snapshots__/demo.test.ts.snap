// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<popconfirm> demo: render [async] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Click To Show
</button>
<!---->"
`;

exports[`<popconfirm> demo: render [basic] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Click To Delete
</button>
<!---->"
`;

exports[`<popconfirm> demo: render [custom] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Discard
</button>
<!---->"
`;

exports[`<popconfirm> demo: render [position] correctly 1`] = `
"<div style=\\"position: relative; width: 440px; height: 280px;\\"><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 0px; left: 70px;\\">
    <!--v-if-->TL
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 0px; left: 180px;\\">
    <!--v-if-->TOP
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 0px; left: 290px;\\">
    <!--v-if-->TR
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 240px; left: 70px;\\">
    <!--v-if-->BL
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 240px; left: 180px;\\">
    <!--v-if-->BOTTOM
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 240px; left: 290px;\\">
    <!--v-if-->BR
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 60px; left: 10px;\\">
    <!--v-if-->LT
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 120px; left: 10px;\\">
    <!--v-if-->LEFT
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 180px; left: 10px;\\">
    <!--v-if-->LB
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 60px; left: 350px;\\">
    <!--v-if-->RT
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 120px; left: 350px;\\">
    <!--v-if-->RIGHT
  </button>
  <!----><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal button\\" type=\\"button\\" style=\\"position: absolute; top: 180px; left: 350px;\\">
    <!--v-if-->RB
  </button>
  <!---->
</div>"
`;
